ziti/cmd/create/create_config_controller.go

```diff
@@ -253,27 +267,44 @@ func SetWebConfig(data *ControllerTemplateValues) {
 	SetWebIdentityCA(data)
 	SetCtrlAltServerCerts(data)
 }
+
+func SetConsoleConfig(v *ConsoleValues, o *ConsoleOptions) {
+	if o.Location == "DISABLED" {
```

Comment:
This should test for "" instead. the default location should be {{ .ZitiHome }} and not a linux-path